<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>489</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Denon Control</string>
  </property>
  <property name="windowIcon">
   <iconset resource="main.qrc">
    <normaloff>:/denon.svg</normaloff>:/denon.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item row="9" column="0">
       <widget class="QLabel" name="lDynVol">
        <property name="text">
         <string>Dyn. Volume</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSlider" name="sVolume">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="lSoundMode">
        <property name="text">
         <string>Sound Mode</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QComboBox" name="cMultiEq">
        <item>
         <property name="text">
          <string>Audyssey</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>By-pass LR</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Flat</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Manual</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Off</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QComboBox" name="cSource">
        <property name="currentIndex">
         <number>5</number>
        </property>
        <item>
         <property name="text">
          <string>Tuner</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Phono</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cd</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dvd</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Net/Usb</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Media Player</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lVolume">
        <property name="text">
         <string>Master volume</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QCheckBox" name="cDynEq">
        <property name="text">
         <string>Dynamic EQ</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="mute">
        <property name="text">
         <string>Mute</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="lEco">
        <property name="text">
         <string>ECO mode</string>
        </property>
       </widget>
      </item>
      <item row="12" column="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lInput">
        <property name="text">
         <string>Input Source</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lSurround">
        <property name="text">
         <string>Surround</string>
        </property>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="QComboBox" name="cDynVol">
        <item>
         <property name="text">
          <string>Off</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Light</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Medium</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Heavy</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="power">
        <property name="text">
         <string>Power</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QComboBox" name="cEco">
        <item>
         <property name="text">
          <string>Off</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>On</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Auto</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lMultiEq">
        <property name="text">
         <string>Multi EQ</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QComboBox" name="cSurround">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>Direct</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Pure Direct</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Stereo</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Standard</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dolby Digital</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dolby Digital NeuralX</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DtsSurround</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Neural</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>WideScreen</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7Ch. Stereo</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SuperStadium</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RockArena</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>JazzClub</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Classic</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MonoMovie</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Matrix</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>VideoGame</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QCheckBox" name="cCinemaEq">
        <property name="text">
         <string>Cinema EQ</string>
        </property>
       </widget>
      </item>
      <item row="11" column="2">
       <widget class="QComboBox" name="cSoundMode">
        <item>
         <property name="text">
          <string>Pure</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Music</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Movie</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Game</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="lDRC">
        <property name="text">
         <string>Dyn. Range</string>
        </property>
       </widget>
      </item>
      <item row="10" column="2">
       <widget class="QComboBox" name="cDRC">
        <item>
         <property name="text">
          <string>Auto</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Low</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mid</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>High</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Off</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>504</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections/>
</ui>
